### ISSUE TITLE
The AHRS algorithms should live in their own repository

### ISSUE THREAD

**Issue by brettaufheber** [adafruit/Adafruit_AHRS#33]
This repository mixes two different things. Firstly, it contains platform-independent AHRS algorithms. Secondly, it integrates with tons of sensor drivers, enhancing the user experience with these drivers. However, anyone who is exclusively interested in the AHRS algorithms, the library ships with numerous undesired dependencies on Adafruit sensor drivers.

For those searching for a standalone version of this library, without any Adafruit sensor driver dependencies, I have established a repository, which is scheduled to mirror the original library while purging the dependencies to Adafruit sensor drivers away from the releases. It's my hope that this will be a temporary solution.

https://github.com/brettaufheber/Adafruit_AHRS_Standalone

**Comment by bruceoberg** [comment]
agreed. i was able to use the AHRS code on its own my making a CMakeLists.txt file that build a library just of the various fusion files. seems weird that other sensor specific stuff is part of this project.
